.github/scripts/pr_body_check.py

```
@@ -0,0 +1,87 @@
+
+def validate_template_sections(template_sections: dict[str, str], pr_sections: dict[str, str]) -> list[str]:
+    """Validate PR sections against template sections and return incomplete sections."""
+    incomplete_sections: list[str] = []
+    for section, template_text in template_sections.items():
+        # Skip the Unit Testing section entirely
+        # until we have a better way to handle it, including the policy itself
+        if "unit testing" in section.lower():
+            continue
+        pr_text = pr_sections.get(section, '')
+        if not normalize(pr_text) or normalize(pr_text) == normalize(template_text):
+            if not is_na(pr_text):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)